symbols/overlay29.yml

```diff
@@ -9669,3 +9757,27 @@ overlay29:
       length:
         EU: 0x4
       description: "[Runtime] Pointer to the dungeon fades struct that maintains the status of screen fades in dungeon mode."
+    - name: AI_THROWN_ITEM_ACTION_CHOICE_COUNT
+      address:
+        EU: 0x23542FC
+      length:
+        EU: 0x4
+      description: "[Runtime] Used to store the number of positions output by GetPossibleAiArcItemTargets and the number of directions/probabilities output by GetPossibleAiThrownItemDirections."
+    - name: AI_THROWN_ITEM_DIRECTION_IS_USED
+      address:
+        EU: 0x237D5D0
```

Comment:
These look to be failing the checks. I think this belongs in `ram.yml`?

Reply:
Moved it

Reply:
The counter actually seems to be located in overlay29, which is weird.